api/src/main/java/org/pmiops/workbench/config/CacheAccessModuleConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package org.pmiops.workbench.config;
+
+import com.google.common.base.Supplier;
```

Comment:
this can be the standard `java.util.function.Supplier` instead

Reply:
It does not work here. `Suppliers.memoizeWithExpiration` is only available in Guava...

Reply:
right - I mean here specifically: because `com.google.common.base.Supplier` inherits from `com.google.common.base.Supplier`.

But it is not so important.

Reply:
AH, I see what you mean. Done